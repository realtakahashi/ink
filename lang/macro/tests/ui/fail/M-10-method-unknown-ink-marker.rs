use ink_lang as ink;

#[ink::contract(version = "0.1.0")]
mod unknown_method_ink_marker {
    #[ink(storage)]
    struct UnknownMethodInkMarker {}

    impl UnknownMethodInkMarker {
        #[ink(constructor)]
        fn constructor() -> Self {
            Self {}
        }

        #[ink(message)]
        fn message(&self) {}

        #[ink(unknown_marker)]
        fn method(&self) {}
    }
}

fn main() {}
