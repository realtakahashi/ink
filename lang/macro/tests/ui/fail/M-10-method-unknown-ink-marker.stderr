error: unknown ink! marker
  --> $DIR/M-10-method-unknown-ink-marker.rs:17:14
   |
17 |         #[ink(unknown_marker)]
   |              ^^^^^^^^^^^^^^^^
